tests/by-util/test_dd.rs

```diff
@@ -1403,6 +1403,31 @@ fn test_bytes_suffix() {
         .pipe_in("abcdef")
         .succeeds()
         .stdout_only("\0\0\0abcdef");
+    new_ucmd!()
```

Comment:
could you please move that into a dedicated function?
easier to spot the error in case of regression
thanks

Reply:
done!